GPU/GPUTracking/DataTypes/GPUTRDInterfaceO2Track.h

```diff
@@ -77,10 +77,11 @@ class trackInterface<o2::track::TrackParCov> : public o2::track::TrackParCov
 
   GPUdi() void setPileUpDistance(unsigned char bwd, unsigned char fwd) { setUserField((((unsigned short)bwd) << 8) | fwd); }
   GPUdi() bool hasPileUpInfo() const { return getUserField() != 0; }
-  GPUdi() unsigned char getPileUpDistanceBwd() const { return ((unsigned char)getUserField()) >> 8; }
```

Comment:
ok and there was another bug here which always returned zero for `getPileUpDistanceBwd()`, because by casting the 16 unsigned int to an unsigned char the information was always cut away